common/history/characters/0_dvg_lotharingia.txt

```diff
@@ -0,0 +1,162 @@
+﻿CHARACTERS = {
+	c:LOT = {
+		
+		
+		create_character = {
+			ruler = yes
+			noble = yes
+			is_general = yes
+			first_name = "Denis"
+			last_name = "de_Aigle"
+			culture = cu:lotharian
+			female = no
+			commander_rank = default
+			interest_group = ig_armed_forces
+			ideology = ideology_jingoist_leader
+			birth_date = 1785.6.20
+			hq = region_pampas

```

Comment:
Invalid HQ region_pampas in  file: common/history/characters/0_dvg_lotharingia.txt line: 5